recipes-core/images/files/nilrt-base-system-image.postinst

```diff
@@ -17,6 +17,7 @@ grub-editenv /boot/grub/grubenv unset enable_initramfs
 
 # Enable ssh by default when installing SystemLink.
 /usr/local/natinst/bin/nirtcfg -s section=systemsettings,token=sshd.enabled,value=True
+/usr/local/natinst/bin/nirtcfg -s section=systemsettings,token=PersistentLogs.enabled,value=True
```

Comment:
Add a new line before this to separate from ssh, and also add a comment

Reply:
I have added a comment now.